src/main/java/PageObjects/FuelsAndBiocomponentsRequirementsPage.java

```diff
@@ -3,36 +3,44 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
 import java.time.Duration;
 
-public class FuelsAndBiocomponentsRequirementsPage extends BasePage{
-    private HeaderPage headerPage;
-    private FooterPage footerPage;
-    private WebDriverWait webDriverWait;
-
-    private By dieselFuelsRequirementsBtn = By.xpath(".//div[@class='col-8 builder-container__content']/div[@id='faq-simple']/div[2]//a");
-    private By flashPointTableValue = By.xpath(".//div[@class='col-8 builder-container__content']/div[@id='faq-simple']/div[2]/div[@id='collapse2']//table//tr[8]");
+public class FuelsAndBiocomponentsRequirementsPage extends BasePage {
+    private final HeaderPage headerPage;
+    private final FooterPage footerPage;
+    private final WebDriverWait webDriverWait;
+    private final By dieselFuelsRequirementsBtn = By.xpath(".//div[@class='col-8 builder-container__content']/div[@id='faq-simple']/div[2]//a");
+    private final By flashPointTableValue = By.cssSelector("#collapse2 .panel-body table tr:nth-of-type(8)");
+    private final By temperatueValueRow = By.cssSelector("#collapse2 .panel-body table tr:nth-of-type(8)>td:nth-child(2)");
```

Comment:
można by pomyśleć o szukaniu elementów w elemencie: https://www.tutorialspoint.com/finding-an-element-in-a-sub-element-in-selenium-webdriver